packages/apollo-server-express/src/ApolloServer.ts

```diff
@@ -29,9 +30,27 @@ export interface ServerRegistration {
   bodyParserConfig?: OptionsJson | boolean;
   onHealthCheck?: (req: express.Request) => Promise<any>;
   disableHealthCheck?: boolean;
-  gui?: boolean;
+  gui?: ((req: express.Request) => Partial<GuiOptions>) | Partial<GuiOptions>;
 }
 
+export interface GuiOptions {
+  enabled: boolean;
```

Comment:
I don't think we need to include an enabled boolean, provided that we can use a boolean instead of a configuration. subscriptions are a good example of how this configuration is done in the engine constructor.